index.js

```diff
@@ -158,8 +158,9 @@ NYC.prototype._prepGlobPatterns = function (patterns) {
 
 NYC.prototype.addFile = function (filename) {
   var relFile = path.relative(this.cwd, filename)
-  var source = stripBom(fs.readFileSync(filename, 'utf8'))
+  var source = this._readTranspiledSource(path.resolve(this.cwd, filename))
   var instrumentedSource = this._maybeInstrumentSource(source, filename, relFile)
```

Comment:
@jamestalmage reading through the codebase I think that `addAllFiles` is the only method that calls `addFile` -- given this, I'm pretty sure we don't need to worry about a performance hit, and this shouldn't cause trouble for reports.

Mind running this branch against one of your larger codebases and making sure there's no performance hit?